packages/api/src/external/carequality/process-outbound-patient-discovery-resps.ts

```diff
@@ -36,6 +37,15 @@ export async function processOutboundPatientDiscoveryResps({
       return;
     }
 
+    log(`Updating patient demographics`);
+    const patientDemoAugmented: boolean = await augmentPatientDemographics(
```

Comment:
nit: no need to set the const type when it can be easily inferred by the response of the function beinng called.

Reply:
No longer relevant.